MergerLogicUnitTests/Utils/S3UtilsTest.cs

```diff
@@ -69,6 +69,103 @@ public static IEnumerable<object[]> GenGetTileParams()
             });
         }
 
+        // TODO: revert to this test
```

Comment:
?

Reply:
This should be here, the fix is temporary and this is the test for before the fix.